tests/IntegrationTests/IntegrationTestCase.php

```diff
@@ -29,4 +30,17 @@ protected function createMatejInstance(): Matej
 
         return $instance;
     }
+
+    protected function assertResponseCommandStatuses(Response $response, ...$expectedCommandStatuses): void
+    {
+        $this->assertSame(count($expectedCommandStatuses), $response->getNumberOfCommands());
```

Comment:
assertCount should be used instead

Reply:
But assertCount takes "expected" as integer, and compares it to array. Here we take array, count/intersect it to get the expected value, and compare it to whatever the response contains.

Looks counter-intuitive on the first hand, but has correct error reporting (`failed asserting that $response->numberOfCommands() is not expected "3"`)

Reply:
Oh, I see, my fault, sorry :)